docs/docusaurus.config.js

```diff
@@ -60,7 +60,7 @@ const config = {
           },
           {to: '/blog', label: 'Blog', position: 'left'},
           {
-            href: 'https://github.com/turnbasedgames/docs',
+            href: 'https://github.com/turnbasedgames/urturn',
```

Comment:
This isn't related to this change, but it fixes a broken link. `/docs` repo doesn't exist. It should be `/urturn`